ts/IterableInboxMessageDisplay.tsx

```diff
@@ -84,21 +84,29 @@ const IterableInboxMessageDisplay = ({
    const handleHTMLMessage = (event: any) => {
       let URL = event.nativeEvent.data
 
+      let action = new IterableAction("openUrl", URL, "")
+      let source = IterableActionSource.inApp
+      let context = new IterableActionContext(action, source)
+
       if (URL === 'iterable://delete') {
          deleteRow(rowViewModel.inAppMessage.messageId)
-      } else if (Iterable.savedConfig.urlHandler) {
-         let action = new IterableAction("openUrl", URL, "")
-         let source = IterableActionSource.inApp
-         let context = new IterableActionContext(action, source)
-
-         Iterable.savedConfig.urlHandler(event.nativeEvent.data, context)
-      }
+         returnToInbox()
```

Comment:
Add return statements for each if logic if no action is needed after returning to inbox. Otherwise the control may flow to next lines unnecessarily.

Reply:
Yes. I will do so. Good catch I know better.